dev-libs/nlohmann_json/nlohmann_json-3.4.0.recipe

```diff
@@ -0,0 +1,36 @@
+SUMMARY="JSON library for Modern C++"
+DESCRIPTION="One-header JSON library with intuitive syntax"
+HOMEPAGE="https://github.com/nlohmann/json"
+SOURCE_URI="https://github.com/nlohmann/json/releases/download/v3.4.0/json.hpp#noarchive"
```

Comment:
COPYRIGHT, LICENSE, REVISION should be placed here: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines)